desktop_entries/eclipse.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Eclipse
+Comment=Eclipse Integrated Development Environment
+Icon=~/eclipse-installer/eclipse-inst/icon.xpm
```

Comment:
The installation create this path? is so repetitive. if this is it i can edit the eclipse-install script.

Reply:
I don't know if it installed elsewhere, but that would be the default if under ~/

Reply:
The Yapi install eclipse-installer in home directory of user, so the ~ path is right